bin/metrics-cloudfront.rb

```diff
@@ -52,11 +52,10 @@ class CloudFrontMetrics < Sensu::Plugin::Metric::CLI::Graphite
          description: 'AWS Region (defaults to us-east-1).',
          default: 'us-east-1'
 
-  option :metric,
-         description: 'Metric to fetch',
-         short: '-m METRIC',
-         long: '--metric',
-         in: %w[Requests BytesDownloaded BytesUploaded TotalErrorRate 4xxErrorRate 5xxErrorRate]
+  option :metrics,
+         description: 'Commas separated list of metric(s) to fetch',
+         short: '-m METRIC1,METRIC2',
+         long: '--metrics METRIC1,METRIC2'
```

Comment:
Are we wanting to keep a 1:1 relationship for distribution ids and metrics to collect? For example if you have no distribution ids defined (meaning all) and we specify `Requests,5xxErrorRate` what would happen? Would it collect all of them for each?

Reply:
It would fetch and retrieve the defined `metrics` for all available CF distributions.